core/src/test/java/com/linecorp/armeria/common/MediaTypeTest.java

```diff
@@ -116,20 +116,21 @@ public void testConstants_areUnique() {
 
     // reflection
     @SuppressWarnings("Guava")
-    private static FluentIterable<Field> getConstantFields() {
-        return FluentIterable.from(asList(MediaType.class.getDeclaredFields())).filter(input -> {
+    static <T, R> FluentIterable<Field> getConstantFields(Class<T> clazz, Class<R>... filterClazz) {
```

Comment:
Though if we can I think it's good to migrate these to return `Stream`, we don't use `FluentIterable` anywhere else in our code

Reply:
> it's good to migrate these to return Stream

+1 😄

Reply:
Mayne better leaving at it is, given the entire test case has been forked from Guava?

Reply:
Ah yeah it's fine to leave it then, can just clean up the `Streams.stream` part.